make gen-check failed

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
make gen-check failed
```
# make gen-check 
go mod tidy -compat=1.17
go clean -testcache
go clean -cache
rm -rf out/linux-amd64
rm -rf out/prom
go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest
go install -a github.com/brancz/gojsontoyaml@latest
go install -a github.com/google/go-jsonnet/cmd/jsonnet@latest
rm -rf manifests/profiles/prom/
mkdir -p manifests/profiles/prom/
mkdir -p out/prom
cp manifests/jsonnet/kube-prometheus.jsonnet out/prom
hack/gen-prom.sh out/prom v0.10.0 kube-prometheus.jsonnet
begin to generate prom manifests
path: out/prom
version: v0.10.0
jsonnet file: kube-prometheus.jsonnet
~/go/src/github.com/kurator-dev/kurator/out/prom ~/go/src/github.com/kurator-dev/kurator
hack/gen-prom.sh: line 13: jb: command not found
Makefile:65: recipe for target 'gen-prom' failed
make: *** [gen-prom] Error 127
```
Solutions:
we should `export PATH=$PATH:$GOPATH/bin` after we install jb
https://github.com/kurator-dev/kurator/blob/33fef92a4a7386d6706a4de1366f5bbf96c2215d/Makefile#L57-L61

or we can use the function defined in util.sh
https://github.com/kurator-dev/kurator/blob/33fef92a4a7386d6706a4de1366f5bbf96c2215d/hack/util.sh#L13-L20
but i don't know how to run functions of shell script in Makefile

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kurator version:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -54,14 +54,8 @@ fix-copyright:
 golangci-lint:
 	hack/golangci-lint.sh
 
-.PHONY: install-tools
-install-tools: 
-	go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest
-	go install -a github.com/brancz/gojsontoyaml@latest
-	go install -a github.com/google/go-jsonnet/cmd/jsonnet@latest
-
 .PHONY: gen-prom
-gen-prom: clean install-tools
+gen-prom: clean
 	rm -rf ${PROM_MANIFESTS_PATH}
 	mkdir -p ${PROM_MANIFESTS_PATH}
 	mkdir -p ${PROM_OUT_PATH}

--- a/hack/gen-prom.sh
+++ b/hack/gen-prom.sh
@@ -4,6 +4,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+
+source "${REPO_ROOT}/hack/util.sh"
+
+# install prerequisite tools
+util::install_tools github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb latest
+util::install_tools github.com/brancz/gojsontoyaml latest
+util::install_tools github.com/google/go-jsonnet/cmd/jsonnet latest
+
 echo 'begin to generate prom manifests'
 echo "path: $1";
 echo "version: $2"

